lxd-profile.yaml

```diff
@@ -0,0 +1,15 @@
+name: juju-default-k8s-deployment-0
+config:
+  linux.kernel_modules: ip_tables,ip6_tables,netlink_diag,nf_nat,overlay
+  raw.lxc: |
+    lxc.apparmor.profile=unconfined
+    lxc.mount.auto=proc:rw sys:rw
+    lxc.cap.drop=
+  security.nesting: true
+  security.privileged: true
+description: ""
+devices:
+  aadisable:
+    path: /dev/kmsg
+    source: /dev/kmsg
+    type: unix-char
```

Comment:
We don't need to match the profile [from the wiki](https://github.com/charmed-kubernetes/bundle/wiki/Deploying-on-LXD#the-profile)? We're missing aadisable1, aadisable2, and boot.autostart.

Reply:
Nope, so these keys in question were failing to validate with juju even when being [deployed on a non-lxd cloud](https://bugs.launchpad.net/juju/+bug/1836226). So they were removed, and the stripped down profile was tested againsted the validation tests in the Jenkins repo. :smile: